rero/rero-ils-ui
# routing: implement new route collection with classes

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- To implement some classes route for the new system of route collection.

## How to test?

- Check All elements on admin frontend
- Use the dev branch on rero-ils
- `bootstrap` and `setup`

## Code review check list
